### PR TITLE
Tom

### PR DESCRIPTION
**Pull request by Zyrck56**
Expected values not reflected on graph until go is pushed.
Renamed some more objects and variables.

### CODE REVIEW

File: FESS-GUI/setpassworddialog.cpp [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #include <QDebug>
 #include <QCryptographicHash>
 #include <iostream>
+#include <mainwindow.h>
```
Comment (JCJutson) [inline FESS-GUI/setpassworddialog.cpp]: why is this included?

**Review by JCJutson — CHANGES_REQUESTED** [review]
I think the changes on setpassworddialog.cpp shouldn't be merged. Other than that, it looks good

File: FESS-GUI/setpassworddialog.cpp (hunk 1 above)
Comment (Zyrck56) [reply FESS-GUI/setpassworddialog.cpp]: removed
